libpod/container.go

```diff
@@ -384,7 +403,7 @@ func (c *Container) Init() (err error) {
 
 	c.state.State = ContainerStateCreated
 
-	if err := c.runtime.state.SaveContainer(c); err != nil {
+	if err := c.Save(); err != nil {
 		return errors.Wrapf(err, "error saving container %s state", c.ID())
```

Comment:
Just return err, the same error message here is being returned in Save() as well. This will just print out the same error message twice.